src/models/user.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "comiditapp/api/src/enums"
+
+type User struct {
+	Role           enums.Role `json:"role"`
+	Name           string     `json:"name"`
+	Email          string     `json:"email"`
+	HashedPassword string     `json:"hashedPassword"`
+	Phone          int        `json:"phone"`
+	Address        []Address  `json:"address"`
+	Menu           []Product  `json:"menu"`
```

Comment:
Deberíamos omitir esto para los usuarios con rol cliente no?

```suggestion
	Menu           []Product  `json:"menu"` `json:",omitempty"`
```

Reply:
True, muy buena esa 💯